src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -360,6 +393,9 @@ export class HaIntegrationCard extends LitElement {
           align-items: center;
           padding-right: 5px;
         }
+        .card-actions .documentation {
+          color: var(--primary-text-color);
+        }
```

Comment:
I don't think that we should add another color. Let's just keep it black but instead let's add the "external link" icon next to the text in the option:

![image](https://user-images.githubusercontent.com/1444314/86059868-7ef22000-ba18-11ea-95e6-73fff67cec52.png)

Reply:
Without modifying the color with var(--primary-text-color) the link inherited the blue anchor color. Here is what it looked like before adding modding the color: 
![doclink](https://user-images.githubusercontent.com/1964858/86063087-10bd5580-ba38-11ea-8b01-7b0952cffb83.PNG)

Heres after:
![hassdoc](https://user-images.githubusercontent.com/1964858/86063128-2c286080-ba38-11ea-8c3d-ae6b77de3800.PNG)

I will add the "external link" indicator and move the manifest fetching to the parent. Thank you for the tips.

Reply:
Oh my bad, coloring is ok 👍

Reply:
You can make it more general, as we never want a link in a different color in that list:
```suggestion
        .card-actions a {
          color: var(--primary-text-color);
        }
```